SQF/dayz_code/stringtable.xml

```diff
@@ -15891,7 +15892,7 @@
 		<Key ID="STR_EPOCH_10OZCOPPER_DESC">
 			<English>10oz Copper Bar</English>
 			<German>10oz Kupferbarren</German>
-			<Russian>Медный слиток (10 унций)</Russian>
+			<Russian>Медный слиток (10 унций)./Russian>
```

Comment:
You can do that, just make the change in your repo and commit the change, it will automatically update on here